homeassistant/components/frontend/www_static/polymer/layouts/home-assistant-main.html

```diff
@@ -130,6 +137,11 @@
       filter="{{selected == 'group' ? 'group' : null}}">
     </partial-states>
 
+    <template if="{{selected == 'scripts'}}">
```

Comment:
Instead of doing this you should be able to integrate it in the above rendering of `partial-states` on line 137. We could move the logic of setting the filter into the `selectPanel` function.